templates/zerver/invite_user.html

```diff
@@ -37,8 +37,7 @@ <h3 id="invite-user-label">{% trans %}Invite users to Zulip{% endtrans %}</h3>
             </div>
             <div class="modal-footer">
                 <button class="button exit btn-danger small rounded" data-dismiss="modal">{{ _('Cancel') }}</button>
-                <button id="submit-invitation" class="button small rounded sea-green"
-                data-loading-text="{{ _('Inviting...') }}" type="submit">{{ _('Invite') }}</button>
+                <button id="submit-invitation" class="button small rounded sea-green" data-loading-text="{{ _('Inviting...') }}" type="submit">{{ _('Invite') }}</button>
```

Comment:
what is the change here?

Reply:
I use 'prettify' in sublime so it changed it into a single line. There is no change in the code there